linera-core/src/unit_tests/client_tests.rs

```diff
@@ -1395,15 +1395,16 @@ where
     let mut sender = builder
         .add_initial_chain(ChainDescription::Root(1), Amount::from_tokens(3))
         .await?;
-    assert!(matches!(sender
+    let obtained_error = sender
         .transfer_to_account(
             None,
             Amount::from_tokens(3),
             Account::chain(ChainId::root(2)),
             UserData(Some(*b"I'm giving away all of my money!")),
         )
-        .await,
-        Err(ChainClientError::LocalNodeError(LocalNodeError::WorkerError(WorkerError::ChainError(error)))) if matches!(*error, ChainError::ExecutionError(ExecutionError::SystemError(SystemExecutionError::InsufficientFunding { .. }), ChainExecutionContext::Operation(_)))
+        .await;
+    assert!(matches!(obtained_error,
+                     Err(ChainClientError::LocalNodeError(LocalNodeError::WorkerError(WorkerError::ChainError(error)))) if matches!(*error, ChainError::ExecutionError(ExecutionError::SystemError(SystemExecutionError::InsufficientFunding { .. }), ChainExecutionContext::Block))
```

Comment:
Rustfmt needs a little help with that. :sweat_smile:

Reply:
What do you think I should do?

Reply:
I'd just break the line manually, so it's not that long. Not important, of course!